PulseSampleApp/app/src/main/java/com/ooyala/sample/players/PulsePlayerActivity.java

```diff
@@ -29,180 +23,91 @@
-      playerSkinLayoutController.onPause();
-    }
-    Log.d(LOG_TAG, "Player Activity Paused");
-    if (player != null) {
-      player.suspend();
-    }
-  }
-
-  @Override
-  protected void onResume() {
-    super.onResume();
-    if (playerSkinLayoutController != null) {
-      playerSkinLayoutController.onResume( this, this );
-    }
-    Log.d(LOG_TAG, "Player Activity Resumed");
-    if (player != null) {
-      player.resume();
-    }
-  }
-
-  /**
-   * Listen to all notifications from the OoyalaPlayer
-   */
-  @Override
-  public void update(Observable observable, Object argN) {
-    final String argument = OoyalaNotification.getNameOrUnknown(argN);
-    if (argument == OoyalaPlayer.TIME_CHANGED_NOTIFICATION_NAME) {
-      return;
-    }
-
-    if (argument == OoyalaSkinLayoutController.FULLSCREEN_CHANGED_NOTIFICATION_NAME) {
-      Log.d(LOG_TAG, "Fullscreen Notification received : " + argument + " - fullScreen: " + ((OoyalaNotification)argN).getData());
-    }
-    // Automation Hook: to write Notifications to a temporary file on the device/emulator
-    String text = "Notification Received: " + argument + " - state: " + player.getState();
-
-    // Automation Hook: Write the event text along with event count to log file in sdcard if the log file exists
-    playbackLog.writeToSdcardLog(text);
-
-    Log.d(LOG_TAG, "Notification Received: " + argument + " - state: " + player.getState());
-  }
-
-  /*
-    Extras
-   */
-
-  /**
-   * Create a VideoItem from the bundled information send to this activity.
-   * @return The created {@link VideoItem}.
-   */
-  public VideoItem getVideoItem() {
-    VideoItem videoItem = new VideoItem();
-
-    videoItem.setTags(getIntent().getExtras().getStringArray("contentMetadataTags"));
-    videoItem.setMidrollPositions(getIntent().getExtras().getFloatArray("midrollPositions"));
-    videoItem.setContentTitle(getIntent().getExtras().getString("contentTitle"));
-    videoItem.setContentId(getIntent().getExtras().getString("contentId"));
-    videoItem.setCategory(getIntent().getExtras().getString("category"));
-    videoItem.setContentCode(getIntent().getExtras().getString("embedCode"));
-
-    return videoItem;
-  }
-
-  /** Start DefaultHardwareBackBtnHandler **/
-  @Override
-  public void invokeDefaultOnBackPressed() {
-    super.onBackPressed();
-  }
-
-  @Override
-  public void onActivityResult(int requestCode, int resultCode, Intent data){
-      if((resultCode == RESULT_OK || resultCode == RESULT_CANCELED) && requestCode == CLICK_THROUGH_REQUEST){
-        Log.i("DemoIntegration","Came back from clickthrough");
-        onResume();
-      }
-  }
+public class PulsePlayerActivity extends AbstractHookActivity {
+
+	public final static String getName() {
+		return "Pulse Player";
+	}
+
+	// Your publisher code goes here
+	final String PCODE  = "tlM2k6i2-WrXX1DE_b8zfhui_eQN";
+	final String DOMAIN = "http://ooyala.com";
+
+	private OoyalaSkinLayoutController playerSkinLayoutController;
```

Comment:
Why are you defining this variable if it is already part of the parent? Either use this one or the one in the parent Activity.